app/models/facility_group.rb

```diff
@@ -1,5 +1,10 @@
 class FacilityGroup < ApplicationRecord
   include PatientSetAnalyticsReportable
+
+  include Discard::Model
```

Comment:
this seems to already be coming from ApplicationRecord

Reply:
You are right. I have added the `scope` to `ApplicationRecord` and removed the other changes from `FacilityGroup`.